test/functional/test_framework/messages.py

```diff
@@ -1244,7 +1288,8 @@ def __repr__(self):
         return "msg_inv(inv=%s)" % (repr(self.inv))
 
 
-class msg_getdata():
+class msg_getdata:
+    __slots__ = ("inv",)
```

Comment:
need to modify https://github.com/dashpay/dash/blob/6e0fdc7321e399ec2fdc2874bb930b4cc956348d/test/functional/test_framework/messages.py#L1228-L1229

Reply:
fixed in different commit and brought in with edit